python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py

```diff
@@ -202,6 +221,7 @@ def get_table_objects() -> Dict[str, StaticTableDetails]:
         "trips": trips,
         "calendar": calendar,
         "calendar_dates": calendar_dates,
+        "directions": directions,
```

Comment:
we don't care about the order that directions are loaded in do we?

Reply:
We do not, the table is not used until RT trips records are loaded.